pkgs/misc/ananicy-cpp/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, gcc11Stdenv
+, fetchFromGitLab
+, makeWrapper
+, cmake
+, spdlog
+, nlohmann_json
+, systemd
+}:
+
+gcc11Stdenv.mkDerivation rec {
+  pname = "ananicy-cpp";
+  version = "unstable-2021-10-13";
+
+  src = fetchFromGitLab {
+    owner = "ananicy-cpp";
+    repo = "ananicy-cpp";
+    rev = "6a14fe7353221c89347eddbbcafb35cf5fee4758";
+    sha256 = "sha256-V0QPXC17ZD2c4MK3DAkzoPgKOU5V5BjfQKUk7I6f8WM=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    cmake
+    systemd
```

Comment:
Why is this declared twice (once here and again on `buildInputs`) :thinking: ?

Reply:
accident